nilearn/plotting/surf_plotting.py

```diff
@@ -624,13 +644,11 @@ def plot_surf(surf_mesh, surf_map=None, bg_map=None,
 
     %(bg_on_data)s
         Default=False.
+    %(scale_bg_map)s
+        Default=True.
```

Comment:
```suggestion
        Default=True.
        
        .. versionadded:: 0.9.3.dev
        
```

Reply:
Should I keep the `.dev`?

Reply:
I don't think so

Reply:
I think that using `.dev` is the convention we use until we decide whether the next release is a patch or a minor. The maintainer then have to update them before the release. Is is still the case @ymzayek ?

Reply:
It is, indeed.

Reply:
Ok I kept the `.dev` everywhere :slightly_smiling_face:

Reply:
Yes indeed it's still part of the release process to update those so you can keep them.